plutus-core/testlib/PlutusCore/Generators/Hedgehog/AST.hs

```diff
@@ -17,18 +18,18 @@ module PlutusCore.Generators.Hedgehog.AST
     , genType
     , genTerm
     , genProgram
+    , genNameMangler
     , mangleNames
     ) where
 
 import PlutusPrelude
 
 import PlutusCore
-import PlutusCore.Core.Plated (termConstantsDeep)
```

Comment:
This `termConstantsDeep` doesn't seem to be used anywhere anymore.

Reply:
It's OK, it's a library function, maybe someone will need it for something eventually. It's not expensive to maintain.